master/buildbot/www/hooks/github.py

```diff
@@ -170,7 +187,17 @@ def handle_pull_request(self, payload, event):
 
         log.msg("Received {} changes from GitHub PR #{}".format(
             len(changes), number))
-        return changes, 'git'
+        defer.returnValue((changes, 'git'))
+
+    @defer.inlineCallbacks
+    def _get_commit_msg(self, repo_url, sha):
+        url = '/commits/{}'.format(sha)
+        http = yield httpclientservice.HTTPClientService.getService(
+            self.master, repo_url)
```

Comment:
It's a local var of function... or any example I can refer of setup a fake server?